.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Dataflow_Java11.groovy

```diff
@@ -20,26 +20,40 @@ import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 
 
-PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_Dataflow',
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11',
   'Run Dataflow ValidatesRunner Java 11', 'Google Cloud Dataflow Runner ValidatesRunner Tests On Java 11', this) {
 
   description('Runs the ValidatesRunner suite on the Dataflow runner with Java 11 worker harness.')
 
-  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 270)
+  def JAVA_11_HOME = '/usr/lib/jvm/java-11-openjdk-amd64'
+  def JAVA_8_HOME = '/usr/lib/jvm/java-8-openjdk-amd64'
 
+  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 270)
   publishers {
     archiveJunit('**/build/test-results/**/*.xml')
   }
 
   steps {
+    gradle {
+      rootBuildScriptDir(commonJobProperties.checkoutDir)
```

Comment:
Can we override the [javaVersion](https://github.com/apache/beam/blob/003c89135754aac3f7e80f50523c8f7caa4ffcee/gradle.properties#L29) property instead of splitting these into two commands?

This would lead us to using the Java 11 compiler with a source and class target of version 8 or do we want to explicitly guarantee that we are using the Java 8 compiler?

Reply:
I think we want to guarantee that Beam compiled with 1.8 as per release process will work on Java 11 runtime.

Reply:
I don't fully understand your response.

Reply:
We want to check if the released version of Beam, which is compiled with Java 8, will work when a user with Java 11 environment imports it from Maven and uses it in their code.

In other words: yes, we want to explicitly use the Java 8 compiler.

Reply:
Java 11 compiler when told to use source/target version of 8 will produce Java 8 classes and not Java 11 classes.